test/fit_tests/common/fit_common.py

```diff
@@ -0,0 +1,528 @@
+'''
+Copyright 2016, EMC, Inc.
+
+Author(s):
+George Paulos
+
+OnRack/RackHD Functional Integration Test (FIT) library
+This is the main common function library for OnRack/RackHD FIT tests.
+'''
+
+# Standard imports
+import os
+import sys
+import json
+import subprocess
+import time, datetime
+import unittest
+import signal
+import re
+import requests
+import pexpect
+import shutil
+
+# Globals
+
+# Pull arguments from environment into ARGS_LIST
+ARGS_LIST = \
+    {
+    "v": os.getenv("VERBOSITY", "0"),
+    "config":  os.getenv("CONFIG", "config"),
+    "stack": os.getenv("STACK", "None"), # Stack label
+    "ora": os.getenv("ORA", "localhost"), # Appliance IP or hostname
+    "bmc": "None", # BMC IP or hostname
+    "sku": os.getenv("SKU", "all"), # node SKU name
+    "obmmac": os.getenv("OBMMAC", "all"), # node OBM MAC address
+    "nodeid": os.getenv("NODEID", "None"), # node ID
+    "hyper": "None", # hypervisor address
+    "version": os.getenv("VERSION", "onrack-devel"), # code version
+    "xunit": os.getenv("XUNIT", False), # XUNIT output
+    "list": os.getenv("LIST", False), # list tests
+    "group": os.getenv("GROUP", "all"), # test group
+    "http": os.getenv("HTTP", "False"), # force http api protocol
+    "https": os.getenv("HTTPS", "False"), # force https api protocol
```

Comment:
I don't think using the environment to store the argument is a good design, especially in consideration of using some common words, such as "VERSION", "CONFIG", "LIST", "HTTP", .etc.

I'd suggest creating a file to store the default value, something like the .vimrc for vim.